pyproject.toml

```diff
@@ -68,7 +68,7 @@ dependencies = [
     "click~=8.0.4",
     "requests>=2.31.0",
     "python-dotenv>=1.0.0",
-    "jf-ingest==0.0.10",
+    "jf-ingest>=0.0.24",
```

Comment:
I'd prefer if this was locked into 0.0.24, rather than 0.0.24 and above. Ingest is getting rolled out bit by bit and I'd prefer that we lock Agent and D.I. into confirmed working versions, rather than somebody risk updating JF ingest and not making necessary changes on the agent side to match it